tests/e2e/navigation/hamburger-menu.js

```diff
@@ -0,0 +1,42 @@
+const commonElements = require('../../page-objects/common/common.po.js');
+const utils = require('../../utils');
+const helper=require('../../helper');
+
+describe('Hamburger Menu tests : ', () => {
+
+  beforeEach(utils.beforeEach);
+
+  it('should open Configuration wizard', () => {
+    helper.waitUntilReady(element(by.css('#update-available')),60000);
+    helper.handleUpdateModal();
+    commonElements.openMenu();
+    commonElements.checkConfigurationWizard();
+  });
+
+  it('should open Guided tour', () => {
+    commonElements.openMenu();
+    commonElements.checkGuidedTour();
+  });
+
+  it('should open About', () => {
+    commonElements.openMenu();
+    commonElements.checkAbout();
+  });
+
+  it('should open User settings', () => {
+    commonElements.openMenu();
+    commonElements.checkUserSettings();
+  });
+
+  it('should open Report bug', () => {
+    commonElements.openMenu();
+    commonElements.checkReportBug();
+  });
+
+  it('should open Configuration app', () => {
+    commonElements.goToConfiguration();
+    const display = element(by.css('[ui-sref="display.date-time"]'));
```

Comment:
Please move the element definitions to page object.

Reply:
@newtewt - Restore the selectors and timeouts that were failing due to my environment. Removed hamburger menu (`hamburger-menu.js`) from this PR. Will work on them separately.